src/stability_sdk/animation.py

```diff
@@ -535,42 +568,65 @@ def render(self) -> Generator[Image.Image, None, None]:
                 self.api._generate.engine_id = DEFAULT_MODEL
 
             diffusion_cadence = max(1, int(self.frame_args.diffusion_cadence_series[frame_idx]))
+            self.set_cadence_mode(enabled=(diffusion_cadence > 1))
+            off_cadence_step = (frame_idx - self.diffusion_cadence_ofs) % diffusion_cadence == 0
```

Comment:
`off_cadence_step` seems like a confusing name to me. how about `is_diffusion_frame` or something like that? I've thought about them as diffusion frames and in-between frames.